testing-modules/junit5-annotations/src/test/java/com/baeldung/junit5/autoclosable/AutoClosableExtensionUnitTest.java

```diff
@@ -0,0 +1,21 @@
+package com.baeldung.junit5.autoclosable;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.jupiter.api.AutoClose;
+import org.junit.jupiter.api.Test;
+
+class AutoClosableExtensionUnitTest {
+
+    @AutoClose
+    DummyAutoClosableResource resource = new DummyAutoClosableResource();
+
+    @AutoClose("clear")
```

Comment:
shouldn't we add a test for the DummyClearableResource too?

Reply:
@vinipx The test itself doesn't really matter.. in fact, in the article, I haven't even shown it. 

On the other hand, we can add a test so that it looks a bit more realistic.. because right now it the field is not used anywhere. I'll add it now 👍

Reply:
Correct, it is more for a consistency perspective. Thanks!